testcases/OpTestEEH.py

```diff
@@ -82,8 +93,30 @@ def gather_logs(self):
         c.run_command(cmd)
         c.run_command("dmesg")
 
+    def check_eeh_hit(self):
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        tries = 30
```

Comment:
Reduce this 30 to 10 sec's is fine. as if it hit EEH it will hit immediately , so 10 sec's sufficient.